# Stamped externel interrupt time

Header header
float64 data
int32 count